prime-dotnet-webapi/Services/Clients/Email/SmtpEmailClient.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Net.Mail;
+using System.Threading.Tasks;
+using System.IO;
+using System.Net.Http;
+
+namespace Prime.Services
+{
+    public class SmtpEmailClient : ISmtpEmailClient
+    {
+        private static HttpClient _client;
+        public SmtpEmailClient(HttpClient httpClient)
+        {
+            _client = httpClient ?? throw new ArgumentNullException(nameof(httpClient));
```

Comment:
This doesn't actually use the http client, so you don't need to DI it in here.

Reply:
So would I just add this as a a service instead of a client?